lib/chef/knife/core/windows_bootstrap_context.rb

```diff
@@ -75,13 +65,18 @@ def config_content
           client_rb = <<~CONFIG
             chef_server_url  "#{chef_config[:chef_server_url]}"
             validation_client_name "#{chef_config[:validation_client_name]}"
-            file_cache_path   "#{ChefConfig::Config.var_chef_dir(true)}/cache"
-            file_backup_path  "#{ChefConfig::Config.var_chef_dir(true)}/backup"
-            cache_options     ({:path => "#{ChefConfig::Config.etc_chef_dir(true)}/cache/checksums", :skip_expires => true})
```

Comment:
`cache_options` was ignoring the user's input and had a subtle path separator change that made the test fail. It is also marked deprecated so it seems bad to include it in new code: https://github.com/chef/chef/blob/master/chef-config/lib/chef-config/config.rb#L804-L813

Reply:
Discussed this in slack and the consensus was this is probably a bug.